src/inverseRobot.js

```diff
@@ -24,7 +24,26 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const values = Object.values(robot);
+  const newRobot = {};
+
+  for (let i = 0; i < values.length - 1; i++) {
+    for (let j = i + 1; j < values.length; j++) {
+      if (values[i] === values[j]) {
+        return null;
+      }
+
+      if (values[i] !== values[j]) {
+        continue;
+      }
+    }
+  }
+
+  for (const key in robot) {
+    newRobot[robot[key]] = key;
+  }
```

Comment:
```suggestion
  for (const key in robot) {
    if (newRobot.hasOwnProperty(robot[key])) {
      return null;
    } else {
      newRobot[robot[key]] = key;
    }
  }
```
Все можна сильно спростити і використати всього лиш один цикл. Фактично, нам потрібно пройтись по ключам робота і записувати нове значення, якщо його ще не існує